tests-hivemq-platform-operator/src/integrationTest/java/com/hivemq/helmcharts/single/HelmCustomServiceAccountIT.java

```diff
@@ -118,6 +125,42 @@ void platformCharts_withCreationAndValidationDisabled_withCorrectExternalPermiss
                 platformNamespace);
 
         K8sUtil.waitForHiveMQPlatformStateRunning(client, platformNamespace, PLATFORM_RELEASE_NAME);
+
+        // assert that the ServiceAccount and permissions are working
+        assertPlatformPodAnnotations();
+    }
+
+    @Test
+    @Timeout(value = 5, unit = TimeUnit.MINUTES)
+    void platformCharts_withCreateAndValidateDisabled_hivemqRunning() throws Exception {
+        helmChartContainer.installOperatorChart(OPERATOR_RELEASE_NAME,
+                "--set",
+                "hivemqPlatformServiceAccount.create=false",
+                "--set",
+                "hivemqPlatformServiceAccount.validate=false",
+                "--set",
+                "hivemqPlatformServiceAccount.permissions.create=false",
+                "--set",
+                "hivemqPlatformServiceAccount.permissions.validate=false",
+                "--namespace",
+                operatorNamespace);
+        helmChartContainer.installPlatformChart(PLATFORM_RELEASE_NAME,
+                "--set",
+                "nodes.serviceAccountName=" + SERVICE_ACCOUNT_NAME,
+                "--set",
+                "nodes.replicaCount=1",
+                "--namespace",
+                platformNamespace);
+
+        // create the ServiceAccount and permissions correctly
+        K8sUtil.createServiceAccount(client, platformNamespace, SERVICE_ACCOUNT_NAME);
+        createRole();
+        createRoleBinding();
```

Comment:
Should we create these before actually installing the operator and the platform, just avoid adding some noise in the logs?

Reply:
In this scenario we have no validation of the ServiceAccount or permissions. So we would never go into an `ERROR` state, even if the resources are not there.

This is more a test that the operator works without the RBAC permissions, that we don't create when all `hivemqPlatformServiceAccount` flags are disabled.